.github/workflows/ci.yml

```diff
@@ -206,7 +208,7 @@ jobs:
               Please submit a PR from the base repository if you are modifying l10n extraction scripts.
           EOF
           else
-            if [[ "$is_default_branch" == 'true' && "$is_push" == 'true' ]]; then
```

Comment:
Kinda weird edge case but if this is a PR from the default branch of the developer (e.g. PR from `master`) we still want the `dry-run`, so I think we should keep this ?

Reply:
Fair.

Reply:
When we originally had the condition to push only it made no sense to check true == true but now we have PR back it makes sense